README.md

```diff
@@ -185,6 +187,10 @@ random:
    seed: -1 # A value of -1 will result in a seed derived from the system clock value
 
 input:
+  # Random data collection
+  random:
+   enabled: false
+   endMax: 1000
```

Comment:
Add a comment here describing how endMax works

Reply:
Comment added. I've also renamed endMax to maxFrames. I think this is clearer.